silkworm/rpc/core/evm_executor.cpp

```diff
@@ -238,9 +238,6 @@ ExecutionResult EVMExecutor::call(
         evm.add_tracer(*tracer);
     }
 
-    if (!txn.sender()) {
-        return {std::nullopt, txn.gas_limit, Bytes{}, "malformed transaction: cannot recover sender"};
-    }
```

Comment:
I would prefer not to remove this check because at the `EVMExecutor` API level it still makes a lot of sense.

Reply:
ok